docs/release-notes/platform_10-9-0_releasenotes.md

```diff
@@ -0,0 +1,124 @@
+---
+id: v10.9.0
+title: Version 10.9.0 Release Notes
+sidebar_label: v10.9.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 04, 2023_
+
+## Console
+
+### New Features
+
+#### Selection of default providers for the Company's projects
+
+Within the Providers section in the Company Overview area it is now possible to find a new feature that allows the user with Company Owner permissions to be able to set the Default providers that will use all the new projects created on the respective company.
+
+![Default Providers Selection](./img/v10.9_defaultProvider.png)
+
+This will enhance the governance for every Company Owner for company's projects and give better control over the management of company tools.
+The provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed in any moment via CMS.
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx).
+
+#### GitHub Actions as CI/CD tool in Console
+
+Starting with this version, the console supports GitHub Actions as a CI/CD tool.
+
+Within the Providers section accessible through the Company Overview area you can now configure a GitHub provider as a CI/CD tool and using it for the deployment of your workload.
+
+For more information, [go to the documentation](/development_suite/deploy/.mdx).
```

Comment:
```suggestion
For more information, [go to the documentation](/development_suite/deploy/deploy.mdx).
```
This link is broken. Did we mean the suggested one?
![image](https://user-images.githubusercontent.com/103102532/235952427-c282f294-f97c-4767-b62f-07f27cb90fb8.png)